downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1562 @@
+        - [Rename System Module to OS](#rename-system-module-to-os)
+        - [Runtime Module Improvements](#runtime-module-improvements)
+        - [Email Module Improvements](#email-module-improvements)
+        - [WebSub Module Improvements](#websub-module-improvements)
+        - [Introduced New Modules](#introduced-new-modules)
+        - [Removed Modules](#removed-modules)
+    - [Code to Cloud](#code-to-cloud)
+    - [Observability](#observability)
+    - [Breaking Changes](#breaking-changes)
+        - [Language](#language)
+    - [Taint Analyzer Update](#taint-analyzer-update)
+
+#### Updating Ballerina
+
+You can use the update tool to update to Ballerina Swan Lake Alpha as follows.
+
+##### For Existing Users
+
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/swan-lake/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
+                        
+> `bal update`
+
+ Next, execute the command below to update to Swan Lake Alpha.
+
+ > `bal dist pull alpha`                 
+
+However, if you are using a Ballerina version below 1.1.0, install via the [installers](/downloads/#swanlake).
+
+##### For New Users
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+#### Highlights
+
+- The `ballerina` command is renamed to `bal`
+    Now onwards, all the commands will start with `bal` E.g., `bal -v`, `bal build`, `bal dist list`.
+- Introduction of hierarchical package names
+    Now, the package name can take the form of `package-name := identifier(.identifer)*`.
+- Introduction of the `Dependencies.toml` file
+- Support for intersection types with errors
+- Support for passing closed records as rest arguments in function/method calls
+- Support for the empty XML value
+- Improvements to the `match` statement
+- Support for cyclic union type descriptors
+- Changes to casting and `toString`/`toBalString` with errors
+- Changes to object-type and record-type inclusions
+- Improved listener declaration
+- Improvements to the lang library modules 
+- Improvements to the `configurable` feature
+- Improvements to the Developer Tools such as the Language Server, Project API, Debugger, Test Framework, Bindgen Tool, Maven Resolver, and Documentation
+- Introduction of REPL support for Ballerina via the `bal shell` command
+- Improvements to the HTTP, WebSocket, gRPC, security, GraphQL, Kafka, NATS, NATS Streaming, RabbitMQ, Time, Runtime, Email, and WebSub standard library modules
+- Rename the System standard library module to OS
+- Introduction of the new Random, RegEx, TCP, UDP, and WebSubHub standard library modules
+- Removal of the Config, Math, Stringutils, and Socket standard library modules
+- VSCode plugin support for code actions and code completions on Code To Cloud
```

Comment:
Will this make sense to an end user? How about "Code action and code completion support for Code to Cloud libraries in the VS Code plugin"?

Reply:
Updated.